workflows/issue_manager.yml

```diff
@@ -0,0 +1,41 @@
+name: WhyDRS Issue Manager  # Name of the workflow
+
+on:
+  schedule:
+    - cron: '0 0 * * *'    # Schedule the workflow to run once a day at midnight UTC
+  workflow_dispatch:        # Allow manual triggering of the workflow from the GitHub UI
+
+jobs:
+  update-project:
+    runs-on: ubuntu-latest  # The workflow runs on the latest Ubuntu runner
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3  # Checks out the repository to the runner
+
+      - name: Set up Python
+        uses: actions/setup-python@v4  # Sets up the Python environment
+        with:
+          python-version: '3.x'        # Specifies the Python version to use
+
+      - name: Install Dependencies
+        run: |
+          pip install PyGithub  # Installs the PyGithub library needed for the script
+
```

Comment:
Or are the comments a little more aligned with making updates down the road simpler? ✏️

Reply:
This also seems unnecessary, as the code is descriptive enough. Thanks for pointing this out!